.scalafix.conf

```diff
@@ -18,3 +18,4 @@ OrganizeImports {
   removeUnused = false
   targetDialect = Scala3
 }
+OrganizeImports.targetDialect = Scala3
```

Comment:
Isn't this duplicating line 19?

Reply:
this was auto-added by metals, sorry about this. Yes, this looks very redundant, I'll revert it